packages/create-endline-app/src/lib/create-endline-app.ts

```diff
@@ -95,6 +95,8 @@ async function createProjectFiles({
   const dependencies = ['endline']
   await installDependencies(packageManager, dependencies)
 
-  const devDependencies = ['typescript', '@types/node']
-  await installDependencies(packageManager, devDependencies, true)
+  if (typescript){
```

Comment:
I think the lint job is complaining about the missing space before the `{`.

Reply:
My bad. Should be good now